js/packages/example-react-native-app/android/app/src/main/java/com/exampleapp/MainActivity.java

```diff
@@ -0,0 +1,48 @@
+package com.exampleapp;
+
+import com.facebook.react.ReactActivity;
+import com.facebook.react.ReactActivityDelegate;
+import com.facebook.react.ReactRootView;
+
+public class MainActivity extends ReactActivity {
```

Comment:
Very nitpicky: could have all the example app classes in kotlin too 😁

Reply:
Yeah, I think we should push for that if we can! Could be next PR, but converting is kind of a pain.

Reply:
This is how little Java/Kotlin I know. I literally can't tell the difference.

Reply:
We can help with that!